form.js

```diff
@@ -0,0 +1,76 @@
+/*global alert ,document*/
+    // store DOM elements in myElements object
+var myElement = {
+    
+    closeForm: document.getElementById('closeForm'),
+	formBody: document.getElementById('formBody'),
+	yes: document.getElementById('yes'),
+	no: document.getElementById('no'),
+	backSpan: document.getElementById('back'),
+	nav: document.getElementById('nav-id'),
+	container: document.getElementById('container-id'),
+	getBack: document.getElementById('get-back'),
+    userNameId: document.getElementById('userName-id'),
+    userAgeId: document.getElementById('userAge-id'),
+    submit: document.getElementById('submit'),
+    spinner: document.getElementById('spinner')
+};
+
+//  go to the "Go Home" page
+myElement.no.onclick = function () {
+    'use strict';
```

Comment:
Once you wrap the whole file with the lambda, you don't need to add `'use strict';` everywhere, only once in the wrapping function.